### ISSUE TITLE
Operational issues

### ISSUE THREAD

**Issue by luha1-1** [wjsilva19/ig_cbir#1]
Hello, I have a question for you. What is the file pleural_all_AP.pickle and how do I generate it?

**Comment by wjsilva19** [comment]
Hello, I will try to update the GitHub in the following weeks with all the remaining code and also upload the pre-trained models. Answering your question directly: to generate pleural_all_AP, I have selected all images that have an AP view and a pleural effusion label that is not uncertain. The reason to select only AP view images is due to most images from the validation set (which was used in this work as the test set) had been acquired with this view. By recommendation of the radiologist, I did not mix PA and AP views.

**Comment by luha1-1** [comment]
Thank you and Look forward to your updates!
